acceptance-tests/src/test/java/gov/di_ipv_drivingpermit/pages/WelshLangDrivingLicencePageObject.java

```diff
@@ -0,0 +1,1015 @@
+
+    @FindBy(xpath = "//*[@id=\"main-content\"]/div/div/div[1]/div[2]/p[1]")
+    public WebElement checkBannerDVAMsg;
+
+    @FindBy(xpath = "//*[@id=\"header\"]")
+    public WebElement dvlaPageHeadingWelsh;
+
+    @FindBy(xpath = "/html/body/div[2]/div/p/strong")
+    public WebElement betaBannerWelsh;
+
+    @FindBy(className = "govuk-phase-banner__text")
+    public WebElement betaBannerWelshSentence;
+
+    @FindBy(xpath = "//*[@id=\"error-summary-title\"]")
+    public WebElement thereIsaProblemText;
+
+    @FindBy(xpath = "//*[@id=\"main-content\"]/div[1]/div/ul/li[1]/a")
+    public WebElement invlidDOBErrorText;
+
+    @FindBy(xpath = "//*[@id=\"main-content\"]/div[1]/div/ul/li[2]/a")
+    public WebElement invlidDLErrorText;
+
+    @FindBy(id = "licenceIssuerRadio")
+    public WebElement radioBtnDVLA;
+
+    @FindBy(id = "licenceIssuerRadio-DVA")
+    public WebElement radioBtnDVA;
+
+    @FindBy(id = "licenceIssuerRadio-noLicence-label")
+    public WebElement noDLOption;
+
+    @FindBy(id = "licenceIssuerRadio-noLicence")
+    public WebElement noDLRadioBtn;
+
+    @FindBy(id = "submitButton")
+    public WebElement CTButton;
+
+    public WelshLangDrivingLicencePageObject() {
+        PageFactory.initElements(Driver.get(), this);
+    }
+
+    public void navigateToDrivingLicenceCRI(String environment) {
+        visitCredentialIssuers.click();
+        assertURLContains("credential-issuers");
+        switch (environment) {
+            case "Build":
+                drivingLicenceCDRIBuild.click();
+                break;
+
+            case "Staging":
+                drivingLicenceCDRIStaging.click();
+                break;
+
+            case "Integration":
+                drivingLicenceCDRIIntegration.click();
+                break;
+
+            default:
+                break;
+        }
+    }
+
+    public void betaBanner() {
+        betaBannerWelsh.isDisplayed();
+    }
+
+    public void betaBannerSentenceWelsh() {
+        betaBannerWelshSentence.isDisplayed();
+    }
+
+    public void changeLanguageToWelsh() {
+        String currentURL = Driver.get().getCurrentUrl();
+        String newURL = currentURL + "/?lang=cy";
+        Driver.get().get(newURL);
+    }
+
+    public void drivingLicencePageURLValidationWelsh() {
+        String expectedUrl = "https://review-d.build.account.gov.uk/licence-issuer/?lang=cy";
+        String actualUrl = Driver.get().getCurrentUrl();
+        LOGGER.info("expectedUrl = " + expectedUrl);
+        LOGGER.info("actualUrl = " + actualUrl);
+        Assert.assertEquals(expectedUrl, actualUrl);
+    }
+
+    public void validateDLPageTitleWelsh() {
+        String actualTitle = Driver.get().getTitle();
+        System.out.print("Actual Title:" + actualTitle);
+        String expTitle = "Pwy wnaeth gyhoeddi eich trwydded yrru y DU? – – GOV.UK";
+        if (actualTitle.contains(expTitle)) {
+            LOGGER.info("Pass : directed to Pwy wnaeth gyhoeddi eich trwydded yrru y DU?");
```

Comment:
Should this not be an assert?

Reply:
Changed to Assert.assertEquals(expectedText, actualText)